crates/common_utils/src/events.rs

```diff
@@ -60,6 +66,10 @@ pub enum ApiEventsType {
     PaymentMethodList {
         payment_id: Option<String>,
     },
+    #[cfg(feature = "v2")]
+    PaymentMethodListForPaymentMethods {
```

Comment:
Nit: Do we wanna rename this enum variant, to have a similar name as the endpoint?